docs/resources/onboarding/docker.md

```diff
@@ -54,6 +64,14 @@ This will print the IP address of your local Docker instance. Open up `/python/h
 
 Substitute your IP address for the one that says `192.168.99.100` in this example (but keep `:5432` at the end).
 
+**Docker for Mac (local Python)**
+
+```
+    "docker_database": {
+        "connect_str": "postgresql://codefordc:codefordc@localhost:5432/housinginsights_docker"
+    },
+```
+
 **Troubleshooting note**: If you already have Postgres installed on your machine, it may be using the 5432 port. If you get an error message saying something related to a port in use, you'll need to change the port both here AND in the docker-compose.yml file. 
 
 ### 1b) Day-to-day use
```

Comment:
I think the line 90 below (new numbering) should change to `docker-compose up -d` to allow access to the docker command line, right?